assets/src/edit-story/components/canvas/useCanvasKeys.js

```diff
@@ -113,6 +124,17 @@ function useCanvasKeys(ref) {
             currentSelectedIds?.length === 1
               ? getNodeForElement(currentSelectedIds[0])
               : null;
+
+          // Check if the selection event happends outside the canvas container,
+          // i.e Checklist panel, Document panel... etc.
+          const selectedAnchor = global.getSelection().anchorNode;
+          const inCanvasContainer = isDescendant(selectedAnchor, container);
+          if (!inCanvasContainer) {
```

Comment:
Instead of checking ancestry, why not simply check if there is any text selection i.e. anchorOffset is non-zero?

Reply:
Ah, thanks for sharing! That is exactly what I was looking for!

Reply:
As per my other comment, `anchorOffset === 0` does _not_ mean there is _no_ selection.

Reply:
Yea, maybe check `getSelection().type` or compare `toString()` to empty string instead.